src/firefly/js/visualize/ui/ImageCenterDropDown.jsx

```diff
@@ -0,0 +1,166 @@
+    if (tAry.find( (t) => pointEquals(t,wp))) {
+        tAry= tAry.filter( (t) => !pointEquals(t,wp));
+    }
+    else {
+        if (tAry.length > MAX_TARGET_LEN-1) tAry = tAry.slice(0, MAX_TARGET_LEN-1);
+    }
+    tAry.unshift(wp);
+    const recent= tAry.map((t) => t.toString());
+    dispatchAddPreference(TARGET_LIST_PREF, recent);
+    setRecentAry(recent);
+}
+
+
+function getRecentTargets() {
+    const strAry= getPreference(TARGET_LIST_PREF, []);
+    return strAry.map( (s) => parseWorldPt(s));
+}
+
+
+function  hasTablesWithCoordinates(pv) {
+    const group= getTableGroup();
+    if (!group || !pv) return false;
+    const tAry= Object.values(group.tables);
+    const coordTable= tAry.find( (t) => t);
+    return Boolean(pv && coordTable);
+}
+
+function prettyTarget(wp) {
+    if (wp.objName) {
+        if (wp.resolver) {
+            return (
+                <Fragment>
+                    <span style={{fontWeight:'bold'}}>{wp.objName}</span>
+                    <span style={{fontSize: '80%' }}>
+                        <span style={{fontStyle: 'italic'}}> by </span>
+                        {`${wp.resolver.toString().toUpperCase()}`}
+                    </span>
+                </Fragment>
+            );
+        }
+        else {
+            return ( <span style={{fontWeight:'bold'}}>{wp.objName}</span> );
+        }
+    }
+    else {
+        return ( <span style={{fontWeight:'bold'}}>{`${wp.x}, ${wp.y} ${coordToString(wp.cSys)}`}</span> );
+    }
+}
+
+
+export function ImageCenterDropDown({visRoot:vr, visible}) {
+
+    const pv= getActivePlotView(vr);
+    const plot= primePlot(pv);
+    const coordTables= hasTablesWithCoordinates(pv);
+    const hasSearchTarget= Boolean(get(plot, ['attributes',PlotAttribute.FIXED_TARGET]));
+    const plotId= plot && plot.plotId;
+    const [recentAry, setRecentAry] = useState(getRecentTargets());
+
+    const moveToTarget= (fields) => {
+        const wp=  parseWorldPt(get(fields,'UserTargetWorldPt'));
+        if (!wp) return;
+        dispatchRecenter({plotId, centerPt:wp});
+        addToRecentTargets(wp, setRecentAry);
+    };
+    //   //todo code to call a future marker option
+    // const createMarkerAndMoveToTarget= (fields) => {
+    //     const wp=  parseWorldPt(get(fields,'UserTargetWorldPt'));
+    //     if (!wp) return;
+    //     moveToTarget(fields);
+    //                              todo: the section below might need to be deferred to the plot has moved to the target first
+    //     const cc = CysConverter.make(plot);
+    //     if (cc.pointInPlot(wp)) {
+    //         const title= prettyTarget(wp);
+    //         const drawLayerId = title;
+    //         dispatchCreateMarkerLayer(drawLayerId, title, plotId, true, wp, true);
+    //     }
+    // };
+    //
+
+    const dropDown= (
+        <SingleColumnMenu>
+
+            <ToolbarButton text='Pan by table row' tip='Center selected images position of the highlighted row'
```

Comment:
It looks to me like the original specification that there be a "fiducial region" of the displayed image within which no panning would occur was not, in the end, followed.

I still think this is important to avoid excessive panning and even possibly confusing behavior.

It is also pretty important, now that I'm playing with this, for no panning to occur in response to *clicking on the displayed mark on the image* for a table row.  This produces an unsettling effect, plays poorly with "lock by click", and makes the UX very sensitive to whether the user hits the symbol exactly or just off by a little.

Reply:
On FITS or zoomed in HiPS not panning occurs if the object object in with in 90% of the displayed area. Zoom out hips do pan.  I think this makes a little more sense.  

However, I did not test clicking on the image or HiPS.  I agree that in that case is should _never_ jump. I am fixing that.

Reply:
CCB-D agrees, after experience with the tool, that we should abandon the idea of the "fiducial region".  The behavior of "pan to table row" should be to move the image *only when necessary to make the selected row's symbol visible*.